client/src/pages/automation/projects/ProjectListItem.tsx

```diff
@@ -200,25 +204,86 @@ const ProjectListItem = ({project, remainingTags}: ProjectItemProps) => {
+                                    duplicateProjectMutation.mutate(project.id!)
+                                }
+                            >
+                                Duplicate
+                            </DropdownMenuItem>
+
+                            <DropdownMenuItem
+                                className="text-xs"
+                                onClick={() => setShowWorkflowDialog(true)}
+                            >
+                                New Workflow
+                            </DropdownMenuItem>
+
+                            <DropdownMenuSeparator />
+
+                            <DropdownMenuItem
+                                className="text-xs text-red-600"
+                                onClick={() => setShowDeleteDialog(true)}
+                            >
+                                Delete
+                            </DropdownMenuItem>
+                        </DropdownMenuContent>
+                    </DropdownMenu>
                 </div>
             </div>
 
-            <AlertDialog
-                danger
-                isOpen={showDeleteDialog}
-                message="This action cannot be undone. This will permanently delete the project and workflows it contains."
-                title="Are you absolutely sure?"
-                setIsOpen={setShowDeleteDialog}
-                onConfirmClick={() => {
-                    if (project.id) {
-                        deleteProjectMutation.mutate(project.id);
-                    }
-                }}
-            />
+            <AlertDialog open={showDeleteDialog}>
+                <AlertDialogContent>
+                    <AlertDialogHeader>
+                        <AlertDialogTitle>
+                            Are you absolutely sure?
+                        </AlertDialogTitle>
+
+                        <AlertDialogDescription>
+                            This action cannot be undone. This will permanently
+                            delete the project and workflows it contains..
+                        </AlertDialogDescription>
+                    </AlertDialogHeader>
+
+                    <AlertDialogFooter>
+                        <AlertDialogCancel
+                            onClick={() => setShowDeleteDialog(false)}
+                        >
+                            Cancel
+                        </AlertDialogCancel>
+
+                        <AlertDialogAction
+                            className="bg-red-600"
+                            onClick={() => {
+                                if (project.id) {
+                                    deleteProjectMutation.mutate(project.id);
+                                }
+                            }}
+                        >
+                            Delete
+                        </AlertDialogAction>
+                    </AlertDialogFooter>
+                </AlertDialogContent>
+            </AlertDialog>
```

Comment:
Zašto nam je 11 linija koda postalo 32? Ovaj novi `AlertDialog` izgleda toksično. Isto kai i `Tooltip`. Puno tih shadcn komponenti bi tribalo apstraktirati da primaju propove, a ne da moramo ovako specificirati preciznu strukturu. Ali to je za neku drugu priliku.

Reply:
@kresimir-coko to ces kasnije sređivati. Pomalo prelazimo naa shadcn, a kasnije ces ti to optimizirati